File: src/main/java/com/terminal/db/ProductHbm.java
package com.terminal.db;

import java.util.Set;

public class ProductHbm {
	
	private String idproduct;
	private String description;
	private Integer stock;
	private Float price;
	private Set <Long> sales;
	
	public ProductHbm(){}
	
	public ProductHbm(String id){
		idproduct = id;
	}
	
	public Set<Long> getsales() {
		return sales;
	}
	public void setsales(Set<Long> sales) {
		this.sales = sales;
	}
	public String getidproduct() {
		return idproduct;
	}
	public void setidproduct(String idproduct) {
		this.idproduct = idproduct;
	}
	public String getdescription() {
		return description;
	}
	public void setdescription(String description) {
		this.description = description;
	}
	public Integer getstock() {
		return stock;
	}
	public void setstock(Integer stock) {
		this.stock = stock;
	}
	public Float getprice() {
		return price;
	}
	public void setprice(Float price) {
		this.price = price;
	}
	
	
}

File: src/main/scala/ar/com/pos/db/ProductHbm.scala
package ar.com.pos.db

import java.util

class ProductHbm {
	
	private var _idproduct: String = _
	private var _description: String = _
	private var _stock: Integer = 0
	private var _price: Float = 0
	private var _sales = new util.HashSet[SaleHbm]()

	def getIdproduct = _idproduct
	def getDescription = _description
	def getStock = _stock
	def getPrice = _price
	def getSales = _sales
	
	def setIdproduct (value:String):Unit = _idproduct = value
	def setDescription (value:String):Unit = _description = value
	def setStock (value:Integer):Unit = _stock = value
	def setPrice (value:Float):Unit = _price = value
	def setSales (value: util.HashSet[SaleHbm]):Unit = _sales = value
}